.github/pull_request_template.md

```diff
@@ -10,4 +10,5 @@ Fixes: #
 
 ### Changes
 
-List out all the changes, and then explain the changes made at a higher level
+List out all the changes, and then explain the changes made at a higher level.
+Please include images/screenshots if applicable.
```

Comment:
if you were intending for this to be on it a newline when rendered in md, it isn't